02-getting_started.Rmd

```diff
@@ -27,190 +27,272 @@ The `r-devel` at [R/trunk](https://svn.r-project.org/R/trunk "svn devel source c
+    You will be prompted to select components at this stage: `User installation`, `64-bit User installation`, or `Custom installation`.
     The default option may be chosen for the questions from this step onwards to complete the installation.
+    
+Daily binaries for `r-devel` are made available for [download and installation](https://cran.r-project.org/bin/windows/base/rdevel.html). 
+
+#### From source
+
+Before installing R from source it some programs are needed, as per the [latest documentation](https://cran.r-project.org/bin/windows/base/howto-R-4.2.html):
 
-### Building R and R packages
+1.  [Rtools](https://cran.r-project.org/bin/windows/Rtools/) is the toolchain bundle that you can use to build R base and R packages containing compiled code, on Windows. 
+  The latest [version Rtools42](https://cran.r-project.org/bin/windows/Rtools/rtools42/rtools.html) can be installed using the [Rtools installer rtools42-XXXX-XXX.exe ](https://cran.r-project.org/bin/windows/Rtools/rtools42/files/).
 
-#### What tools do you need to build R from source on Windows?
+2.  A LaTeX compilers is needed to install and building R, check packages and build  manuals. 
+    The MiKTeX distribution of LaTeX that is used on CRAN can be downloaded from <https://miktex.org>. 
+    Once installed open it via the windows start menu.
+    It might ask to check for updates and, most importantly, to make it available in the PATH; accept both.
 
-1.  [RTools](https://github.com/r-windows/docs/blob/master/faq.md#what-is-rtools) is the toolchain bundle that you can use to build R base and R packages containing compiled code, on Windows.
+1.  Open the Rtools42 terminal, and update and get the tools:
 
-2.  You also need a distribution of LaTeX installed for building R and checking packages.
-    The MiKTeX distribution of LaTeX that is used on CRAN can be downloaded from <https://miktex.org>.
+    ```sh
+    pacman -Syuu
+    pacman -Sy wget subversion
+    ```
+    
+3. Retrieve the latest source code via subversion:
 
-#### How to setup `RTools`?
+    ```sh
+    export TOP_SRCDIR="$HOME/Downloads/R"
+    svn checkout https://svn.r-project.org/R/trunk/ "$TOP_SRCDIR"
+    ```
 
-1.  The latest version of `RTools` can be downloaded from <https://cran.r-project.org/bin/windows/Rtools/> and run in the Windows-style installer.
-    You will need to know if you have a 32-bit or 64-bit Windows machine (right-click `This PC` in Windows Explorer and check the properties if you are unsure).
+    If you already have it you should update it with :
 
-2.  Don't forget to add `RTools` to the path as documented on the download page.
+    ```sh
+    cd $TOP_SRCDIR
+    svn update
+    ```
+
+    You can also use a SVN client such as TortoiseSVN (<https://tortoisesvn.net/>, command line tool, and Windows Explorer integration) or SlikSVN (<https://sliksvn.com/download/>, just the command line tool) so that it can be also found by other tools.
```

Comment:
This comes from a previous comment from Heather to install subversion so that Rstudio or any other developer tool can use them. I only used TortoiseSVN and it worked via right click menu in Windows.